### PR TITLE
fix(Popup): restore ability to render popup from React element

### PR DESCRIPTION
**Pull request by kuzhelov**
## TODO
- [x] update changelog

------

With recent changes being introduced to shorthand factories (specifically, because of the change that now dictates to pass React Element shorthand value 'as is'), the following way of defining `Popup` has become broken:

```jsx
<Popup content={<p>This is the content that will be rendered 'as is'</p>} ... />
```

The way it was broken is that now the content of the `<p>` element is, essentially, not visible (however, while still rendered in the DOM tree), because positioning styles are not applied.

----

This corresponds to the following example on the docs page:

image

----

This PR restores this functionality.

### CODE REVIEW

File: src/components/Popup/Popup.tsx [hunk 1]
```diff
@@ -266,16 +269,23 @@ export default class Popup extends AutoControlledComponent<Extendable<PopupProps
...
-    })
+    /**
+     * if there is no focus trap wrapper, we should apply
+     * HTML attributes and positioning to popup content directly
+     */
+    const popupContentAttributes = accessibility.focusTrap ? {} : popupWrapperAttributes
+
+    const popupContent = React.isValidElement(content)
+      ? React.cloneElement(content, {
+          ...content.props,
```
Comment (layershifter) [inline src/components/Popup/Popup.tsx]: Why we need a spread there? React will add missing properties: https://codesandbox.io/s/9ornj4qrko

```suggestion
React.cloneElement(content, popupContentAttributes)
```

**Review by layershifter — CHANGES_REQUESTED** [review]
This issue shows that our idea from #496 may be not so good.

The factories change is not released yet, so I will block this PR for additional discussion on sync.

**Review by layershifter — APPROVED** [review]
Unblocked, let's go.

**Comment by kuzhelov** [comment]
@layershifter 

As we've agreed with @layershifter while having discussion around that, moving with this now but leaving this topic for future discussion around shorthand factories, as this change might serve as a warning sign for the approach we've taken, especially if this pattern 

```
isValidElement() ? clone() : create()   // © @layershifter :)
```

will be repeated for other components in many places.

**Pull request merged**